File: median/median.py
# Starter Code
class Solution:
    def median_array(self,arr1,arr2,n,m):
        median_value=10.9056
        #Write your code here without removing the exising code
	#the variable 'arr1' and 'arr2' contains array of integers of size n and m respectively.
	#modified the double median_value contain the output of the program upto 2 place of decimal.
        return median_value



if __name__ == '__main__':
        n = int(input())
        m = int(input())
        str1 = input()
        str2 = input()
        arr1 = [int(int(i)) for i in str1.strip('[]').split(',')]
        arr2 = [int(int(i)) for i in str2.strip('[]').split(',')]
        x = Solution()
        ans = x.median_array(arr1,arr2,n,m)
        print("%.2f" %ans)

File: median/median.cpp

// Starter Code
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
double median_array(std::vector<int>&arr1,std::vector<int>&arr2,int n,int m)
{
    double median_value =8;
    
    //Write your code here without removing the exising code
	//the variable 'arr1' and 'arr2' contains array of integers of size n and m respectively.
	//modified the double median_value contain the output of the program upto 2 place of decimal.

    return median_value;
}
};

int main()
{
    
    int n,m;
    std::cin>>n>>m;
    std::string s1,s2;
    std::cin >> s1>>s2;
    std::istringstream iss1{std::regex_replace(s1, std::regex{R"(\[|\]|,)"}, " ")};
    std::vector<int> v1 = {std::istream_iterator<int>{iss1}, std::istream_iterator<int>{}};
    std::istringstream iss2{std::regex_replace(s2, std::regex{R"(\[|\]|,)"}, " ")};
    std::vector<int> v2 = {std::istream_iterator<int>{iss2}, std::istream_iterator<int>{}};
    Solution ob;
    double ans = ob.median_array(v1,v2,n,m);
    std::cout << std::fixed;
    std::cout << std::setprecision(2) << ans <<std::endl;
    return 0;
}